src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -95,15 +95,23 @@ extension DynamicTableViewController {
 			view?.layoutMargins = insets
 			return view
 
-		case let .image(image, accessibilityLabel: label, accessibilityIdentifier: accessibilityIdentifier, height, accessibilityTraits):
+		case let .image(image, title, accessibilityLabel: label, accessibilityIdentifier: accessibilityIdentifier, height, accessibilityTraits):
 			let view = tableView.dequeueReusableHeaderFooterView(withIdentifier: HeaderFooterReuseIdentifier.header.rawValue) as? DynamicTableViewHeaderImageView
 			view?.imageView?.image = image
 			if let label = label {
 				view?.imageView?.isAccessibilityElement = true
-				view?.imageView?.accessibilityLabel = label
+				if let title = title {
+					view?.imageView?.accessibilityLabel = title + "\n" + label
+				} else {
+					view?.imageView?.accessibilityLabel = label
+				}
```

Comment:
I think it might make sense to keep both elements separate, first let VoiceOver focus the label and then the image